<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>泵浦模块控制程序</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>控制区</string>
    </property>
    <widget class="QGroupBox" name="groupBox_controlMode">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>控制模式</string>
     </property>
     <widget class="QPushButton" name="btn_rs232">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>RS232</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_local">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>29</y>
        <width>75</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>本地</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_port">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>端口控制</string>
     </property>
     <widget class="QPushButton" name="btn_openPort">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开端口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_serialPort">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_flow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>电流控制</string>
     </property>
     <widget class="QPushButton" name="btn_start">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>启动</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_standBy">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>待机</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>31</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>mA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_maxFlow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>电流最大值：</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_flow</zorder>
    <zorder>groupBox_controlMode</zorder>
    <zorder>groupBox_port</zorder>
   </widget>
   <widget class="QPushButton" name="btn_clearMsg">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>630</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空日志</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_recv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>接收信息：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_flow_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>实时状态</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>实时电流值：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_flowValue">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>291</width>
       <height>30</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>mA</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_send">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>451</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>操作日志：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clearMsg_recv">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>630</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空接收消息</string>
    </property>
   </widget>
   <zorder>groupBox_flow_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>btn_clearMsg</zorder>
   <zorder>textEdit_recv</zorder>
   <zorder>label_7</zorder>
   <zorder>textEdit_send</zorder>
   <zorder>label_8</zorder>
   <zorder>btn_clearMsg_recv</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
